### ISSUE TITLE
save and load the accuracy in model

### ISSUE THREAD

**Issue by shangguan91** [rusty1s/pytorch_geometric#1484]
i have following code to save the model, 

``
```
def` save_model(model_dictionary):
    checkpoint_directory = 'torch_directory'
    file_path = os.path.join(checkpoint_directory, 'model.pt')
    torch.save(model_dictionary, file_path)

```

make a dictionary to save model state and optimizer
```
model_dictionary = {
            'model_state': model.state_dict(),
            'model_optimizer': optimizer.state_dict()
            }
save_model(model_dictionary)
```
`

How can i save specifically test_acc, and calculate the average accuracy over 100 epochs

**Comment by rusty1s** [comment]
I‘m not sure I fully understand the question. If you also want to save additional information like test_acc, why not simply add it to `model_dictionary`?
